### PR TITLE
SPCTR-1581 Price List Presets Fix- Presets overriding image data and columns

### PR DESCRIPTION
**Pull request by aapatre**
### Description
- Selected images were overriden when using/switching presets but now this has been fixed by introducing the 'Show image' toggle
- Switching/using presets was overriding columns, so hence removed the feature for overriding columns using presets since it was inconvenient UX

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by aapatre** [comment]
Changed the logic 
cc: @RajkiranBagal

File: src/blocks/price-list/edit.js [hunk 1]
```diff
@@ -126,6 +126,26 @@ const UAGBRestaurantMenu = ( props ) => {
...
+            .getBlocksByClientId( props.clientId )[0]
+            .innerBlocks.forEach( function( block ) {
+
+                dispatch( 'core/block-editor' ).updateBlockAttributes(
+                    block.clientId, {
+                        showImage: props.attributes.showImage,
+                    }
+                );
+                dispatch( 'core/block-editor' ).updateBlockAttributes(
+                    block.clientId, {
```
Comment (RajkiranBagal) [inline src/blocks/price-list/edit.js:141]: Why are you updating twice?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/price-list/edit.js (hunk 1 above)
Comment (aapatre) [reply src/blocks/price-list/edit.js]: Duplicate code added by mistake, removed it

**Comment by aapatre** [comment]
removed the duplicate code 
cc: @RajkiranBagal

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**